opentelemetry-api/src/opentelemetry/attributes/__init__.py

```diff
@@ -31,7 +30,7 @@
 
 def _clean_attribute(
     key: str, value: types.AttributeValue, max_len: Optional[int]
-) -> Optional[types.AttributeValue]:
+) -> Optional[Union[types.AttributeValue, tuple[str | int | float, ...]]]:
```

Comment:
I don't remember which version `|` was introduced in. But if it's supported, can we use it in place of the outer `Union`?

Reply:
It has been added in 3.10